src/header.iced

```diff
@@ -0,0 +1,90 @@
+{prng} = require('crypto')
+{createHash} = require('crypto')
+msgpack = require('purepack')
+crypto = require('keybase-nacl')
+nonce = require('./nonce.iced')
+
+compute_mac_key = (encryptor, header_hash, pubkey) ->
+  zero_bytes = Buffer.alloc(32)
+  mac_box = encryptor.encrypt({plaintext : zero_bytes, nonce : nonce.nonceForMACKeyBox(header_hash), pubkey})
+  return mac_box.slice(-32)
```

Comment:
I'm just going to create a global variable `crypto_onetimeauth_BYTES = 32` because if we want to use the NaCl lib's constants I'd have to go expose it in node-nacl